src/pybind/mgr/cephadm/module.py

```diff
@@ -108,7 +108,7 @@ def os_exit_noop(status: int) -> None:
 DEFAULT_IMAGE = 'quay.io/ceph/ceph'
 DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.43.0'
 DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.5.0'
-DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:0.0.3'
+DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:v0.0.5'
```

Comment:
change https://github.com/ceph/ceph/blob/main/src/cephadm/cephadmlib/constants.py#L15 as well, it hasn't been getting updated but should have been.

Reply:
Thanks @adk3798, done